add_app/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask import Blueprint
+
+addition_bp =Blueprint('addition_bp',__name__)
+
+from . import  routes
```

Comment:
Please add a space after every file

Reply:
WIP

Reply:
@codetaskmaster kindly review my codes once again, I have implemented the changes. Thanks